app/services/media_storage/azure_adapter.rb

```diff
@@ -2,15 +2,15 @@
 
 module MediaStorage
   class AzureAdapter < AbstractAdapter
-    attr_reader :url, :public_container, :private_container, :storage_account_name, :get_expiration, :put_expiration, :client, :signer
+    attr_reader :url_prefix, :public_container, :private_container, :storage_account_name, :get_expiration, :put_expiration, :client, :signer
 
     DEFAULT_EXPIRES_IN = 3 # time in minutes, see get_expiry_time(expires_in)
 
     def initialize(opts={})
       @storage_account_name = opts[:azure_storage_account]
       @public_container = opts[:azure_storage_container_public]
       @private_container = opts[:azure_storage_container_private]
-      @url = opts[:url]
+      @url_prefix = opts[:url_prefix]
```

Comment:
This change will need some updates to the failing specs, https://travis-ci.org/github/zooniverse/panoptes/jobs/741473601
```
1) MediaStorage::AzureAdapter#get_path when medium is public returns the path as a https link
     # ./spec/services/media_storage/azure_adapter_spec.rb:125:in `block (4 levels) in 

  2) MediaStorage::AzureAdapter#get_path when medium is public uses the StoredPath.media_url method
     # ./spec/services/media_storage/azure_adapter_spec.rb:120:in `block (4 levels) in 
```

As well as the storage initializer https://github.com/zooniverse/panoptes/blob/8f9e1c06cad27829bb931f69bda36dddb51b3da7/config/initializers/storage.rb#L9